CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of grafana.
 
 ## Unreleased
```

Comment:
Can you flip this to 4.0.2 and set today's date please, then we can get this merged and released